system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -83,13 +83,18 @@ typedef long suseconds_t;
 
 
 #if defined(__NEED_pthread_attr_t) && !defined(__DEFINED_pthread_attr_t)
+typedef struct {
+    union {
+        int __i[10];
+        volatile int __vi[10];
+        unsigned __s[10];
+    } __u;
 #ifdef __EMSCRIPTEN__
```

Comment:
why did these shrink from 11 to 10?

Reply:
The 11th field is now the pointer declared below, which will differ in size between 32/64-bit.

I don't think this decl was working as intended anyway, since the pointer was passed in field `9` in the existing code.